app/uk/gov/hmrc/incorporatedentityidentificationfrontend/views/templates/govuk_template.scala.html

```diff
@@ -59,13 +59,13 @@
 
```

Comment:
I created this ticket the other day to update the PageConfig used in the Error templates so I think it would be worth fixing then https://jira.tools.tax.service.gov.uk/browse/SAR-9647